CASC/PortfolioMode.cpp

```diff
@@ -132,8 +133,13 @@ bool PortfolioMode::searchForProof()
 
     //we normalize now so that we don't have to do it in every child Vampire
     ScopedLet<Statistics::ExecutionPhase> phaseLet(env.statistics->phase,Statistics::NORMALIZATION);
-    Normalisation().normalise(*_prb);
     
+    if (env.options->shuffleInput()) { // instead to "combing things into shape" we shuffle and ruffle them
+      Shuffling().shuffle(*_prb);
```

Comment:
Well spotted! This was from one of the very first commits and I didn't want to go through the pains of editing it out. However, I plan a separate PR with my other changes to `PortfolioMode` and strategy related things which will override this... So, since this change is harmless for non-shuffling workflows I guess we can simply ignore its effect for now.